build/build.sh

```diff
@@ -92,6 +92,7 @@ if [ "${BUILD_PDF:-}" != "false" ] && [ -n "$DOCKER_RUNNING" ]; then
     arachnysdocker/athenapdf:2.16.0 \
     athenapdf \
     --delay=${MANUBOT_ATHENAPDF_DELAY:-1100} \
+    --timeout=240 \
```

Comment:
```suggestion
    --timeout=360 \
```
@agitter should I just keep increasing this until the manubot build works? The downside of scite it seems like :(

Reply:
It's be grand if we could disable this in the PDF builds.

Reply:
I may try to toggle scite off and on depending on where we're at in the pipeline? Since it seems like it may be incompatible with appveyor bot (which has a 60 min limit) -- appveyor is really helpful for text development whereas scite is helpful for checking things over as we're wrapping a section up!

Reply:
I still haven't looked closely at the scite integration to see what it does or how to speed it up or cache it.  I'll make a commit to try disabling scite from the athenapdf builds.

I thought about caching more of the Manubot references to speed up builds.  I'm not sure if that will help enough if scite is slow.

Adding a line of whitespace before the `@individual-pathogenesis` alias fixed the arXiv reference warnings in the build log.  I fixed one more typo.

Reply:
I forgot that athenapdf is using the HTML that has already been generated.  We could build a second version of the HTML that doesn't use scite.  However, that might still be really slow.

Should we disable scite for now to see if we can build successfully?  Then we can look into reference caching or other speedups in a separate pull request.